File: Pascal/fizzbuzz2.pas
program FizzBuzz1;
const
	limit = 100;
var
	i: integer;
begin
	for i := 1 to limit do
	begin
		case i mod 15 of
			0: write('FizzBuzz');
			3, 6, 9, 12: write('Fizz');
			5, 10: write('Buzz')
		else
			write(i)
		end;
		if i < limit then write(', ')
	end;
	writeln()
end.

File: Nim/fizzbuzz2.nim
const limit = 100

for i in 1..limit:
  case i mod 15:
    of 0: stdout.write "FizzBuzz"
    of 3, 6, 9, 12: stdout.write "Fizz"
    of 5, 10: stdout.write "Buzz"
    else: stdout.write i
  if i < limit: stdout.write ", "

stdout.write '\n'
